src/main/java/software/bernie/techarium/datagen/TechariumRecipeProvider.java

```diff
@@ -93,4 +99,14 @@ private static void buildBotariumRecipe(Item seed, IItemProvider drop, Ingredien
                         new ResourceLocation(Techarium.ModID,
                                 "botarium/minecraft/" + seed.getRegistryName().getPath()));
     }
+
+    private static void buildExchangeStationRecipe(IItemProvider input, int inputAmount, IItemProvider output, int outputAmount, Consumer<IFinishedRecipe> consumer) {
+        ExchangeStationRecipe.builder()
+                .input(new ItemStack(input, inputAmount))
+                .output(new ItemStack(output, outputAmount))
+                .construct()
+                .build(consumer,
+                        new ResourceLocation(Techarium.ModID,
+                                "exchange_station/" + output.asItem().getRegistryName().getPath()));
```

Comment:
Can we also separate this by namespace of the output item? So if it gives you a thermal item it goes in the /thermal/ folder, etc.